### PR TITLE
Added note about algorithm used to generate keys to HIPAA Eligible Segment 

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
Added note that Segment uses a a RSAES OAEP SHA 256 algorithm to encrypt data to the HIPAA Eligible Segment page

### Merge timing
ASAP once approved!

### Related issues (optional)
#5233

### CODE REVIEW

File: src/privacy/hipaa-eligible-segment.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ These logs can be provided upon request. For specific requests, please reach out
 
 ## Data encryption
 
-Segment encrypts the data in select fields [marked as yellow in the Privacy Portal](/docs/privacy/portal/#default-pii-matchers) before sending them to event stream, cloud mode destinations, further supporting HIPAA compliance in your destinations. 
+Segment encrypts the data in select fields [marked as yellow in the Privacy Portal](/docs/privacy/portal/#default-pii-matchers) before sending them to event stream, cloud mode destinations, further supporting HIPAA compliance in your destinations. Segment's public and private key pairs are generated using a a RSAES OAEP SHA 256 algorithm.
```
Comment (pwseg) [inline src/privacy/hipaa-eligible-segment.md:38]:
```suggestion
Segment encrypts the data in select fields [marked as yellow in the Privacy Portal](/docs/privacy/portal/#default-pii-matchers) before sending them to event stream, cloud mode destinations, further supporting HIPAA compliance in your destinations. Segment's public and private key pairs are generated using a RSAES OAEP SHA 256 algorithm.
```

**Review by pwseg — CHANGES_REQUESTED** [review]
One small fix.

File: src/privacy/hipaa-eligible-segment.md (hunk 1 above)
Comment (pwseg) [reply src/privacy/hipaa-eligible-segment.md:38]: Oops, only just now seeing that this is still a draft!

Comment (forstisabella) [reply src/privacy/hipaa-eligible-segment.md]: s'all good!

**Review by pwseg — APPROVED** [review]

**Pull request merged**